aws/v18.4.2/announcement.md

```diff
@@ -0,0 +1,3 @@
+**Workload cluster release v18.4.2 for AWS is available**. This is a patch release that fixes a problem that caused CNI downtime when upgrading from v18 to v19. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v18.4.2/).
+
+> **_Info:_** Cilium support that was introduced in 18.0.0 was rolled back to AWS CNI because of an unresolved upstream bug. Cilium support is postponed to version v19.0.0.
```

Comment:
maybe we also add, there was an issue with the old sqs and event IAM permissions

***IAM Permissions Requirements***
The minimal requirement for the IAM permissions is [Version 3.3.0](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/blob/master/CHANGELOG.md#330---2023-05-11) of [giantswarm-aws-account-prerequisites](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/) repository.

Reply:
yeah i will add this